pass-deposit-services/deposit-core/src/main/java/org/eclipse/pass/deposit/provider/nihms/NihmsMetadataSerializer.java

```diff
@@ -210,7 +210,40 @@ private void write_metadata(Document doc) {
             }
         }
 
-        // Could add grant information here if it was useful.
-        // Can only be used for a set list of funders
+        List<DepositMetadata.Grant> grantsList = metadata.getGrantsMetadata();
```

Comment:
I agree `write_metadata` is rather long. I added the grants here because it fit with the rest. I didn't want to touch what was already written, but perhaps I could create a method for each component of the XML file: `manuscript`, `article`, `journal` etc? Something along the lines of:

```
private void write_metadata(Document doc) {
    Element root = initializeRootElement(doc);

    addManuscriptMetadata(doc, root);
    addArticleMetadata(doc, root);
    addJournalMetadata(doc, root);
    addManuscriptTitle(doc, root);
    addContacts(doc, root);
    addGrants(doc, root);
}
```

Reply:
I thought about creating a method just for the grants, but it would seem awkward for only that component to have it's own method?

Reply:
Yes, your suggestion would be good clean up.

Reply:
Done.